src/website/layout.gleam

```diff
@@ -0,0 +1,292 @@
+import gleam/dict
```

Comment:
Merge this module back into the page module please.

The fact the `PagaMeta` data type no longer has an appropriate to live without creating dependency cycles shows that this is one module and not two.